src/js/tech/tech.js

```diff
@@ -339,13 +395,13 @@ class Tech extends Component {
    */
 
   textTracks() {
-    this.player_.textTracks_ = this.player_.textTracks_ || new TextTrackList();
-    return this.player_.textTracks_;
+    this.textTracks_ = this.textTracks_ || new TextTrackList();
```

Comment:
Interesting. I guess that's another way of doing it :). I'll take a closer look at this tomorrow.